src/components/vuestic-components/va-badge/VaBadge.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <div
+    class="va-badge"
+   :class="badgeClass"
+  >
+    <div
+      class="va-badge__badge mr-2"
+      :style="badgeStyle"
+      v-if="!isEmpty"
+    >
+      <div class="va-badge__badge__content">
```

Comment:
What's the reason to have so many nesting levels?

Reply:
Vuetify have a similar realization badge component
Example: https://vuetifyjs.com/en/components/badges#badges

Reply:
@Kreezag I see only 2 levels of nesting in there.